Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -2766,6 +2766,14 @@ extension Data {
         public static let endLineWithCarriageReturn = Base64EncodingOptions(rawValue: 1 << 4)
         /// When a maximum line length is set, specify that the line ending to insert should include a line feed.
         public static let endLineWithLineFeed       = Base64EncodingOptions(rawValue: 1 << 5)
+
+        /// Use the base64url alphabet to encode the data
+        @available(FoundationPreview 6.2, *)
```

Comment:
We'll need to update this to 6.3 since we've passed 6.2. Just so we don't forget we'll want to update the proposal too